File: lc1313.js
var decompressRLElist = (nums)=> {
    let n = nums.length;
    let ans = [];
    for (let i=0; i<n; i=i+2){
        for (let j = 0; j < nums[i]; j++) {
            ans.push(nums[i + 1])
        }
    }
    return ans;
};

File: py/lc1313.py
def decompressRLElist(nums):
    n = len(nums)
    ans = []
    for i in range(0, n-1, 2):
        ans+= [nums[i+1]] * nums[i]
    return ans

nums = [1,2,3,4]
print(decompressRLElist(nums))
